src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+
+function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`https://news-api-ibvn.onrender.com/api/articles`)
```

Comment:
This does work, but would be much neater if you had it as a function in an api.js file